t/26-video_stream.t

```diff
@@ -205,6 +205,27 @@ subtest 'input events' => sub {
         "ctrl-x\n",
         "s\n", "o\n", "m\n", "e\n", "spc\n", "t\n", "e\n", "s\n", "t\n", "ret\n",
     ], "correct commands sent";
+
+    $bmwqemu::vars{GENERAL_HW_KEYBOARD_URL} = 'http://127.0.0.42:42000/cmd';
+
+    # mock ua
+    my $urls = [];
+    my $user_agent_mock = Test::MockModule->new('Mojo::UserAgent');
+    my $http = Test::MockModule->new('Mojo::Transaction::HTTP');
+    my $req_mock = Test::MockModule->new('Mojo::Message::Request');
```

Comment:
Thanks for updating tests. This variable is unused now. Also ensure that the modules `Mojo::Transaction::HTTP` and `Mojo::Message::Response` are properly imported (we should not rely on accidental implicit imports).